ext/net/lib.deno_net.d.ts

```diff
@@ -138,6 +138,8 @@ declare namespace Deno {
 
     readonly readable: ReadableStream<Uint8Array>;
     readonly writable: WritableStream<Uint8Array>;
+
+    readonly fd: number;
```

Comment:
I don't think this is desirable addition to Deno API because no other resources expose file descriptor.

We generally use rid as a substitute to fd in node compat layer. Doesn't it work to substitute `conn.rid` to `handle.fd` for TCP handle for your use case?

Reply:
Hi thanks for the review, since the rid is a pointer to the internal tcpstream resource and is an abstraction, we would like to print the actual file descriptor the connection is using for observability purposes. Thank you

Reply:
@ry @bartlomieju What do you think?

Reply:
I'm fine exposing `fd` in the Node compat API, but we shouldn't do it in the `Deno` APIs. IMO this should be handled the same way we're passing resource IDs between Deno APIs and Node APIs (via a private symbol)

Reply:
@bartlomieju Got it, I pushed another commit to use internal symbols to expose fd, thank you
https://github.com/denoland/deno_core/pull/1061